### ISSUE TITLE
Error: aws_elasticache_cluster.xxxxxx: "port": required field is not set

### ISSUE THREAD

**Issue by aprice** [terraform-providers/terraform-provider-aws#4149]
### Terraform Version
```
Terraform v0.11.6
+ provider.aws v0.1.4
```

### Affected Resource(s)
aws_elasticache_cluster

### Expected Behavior
Per the documentation:

> port – (Optional) The port number on which each of the cache nodes will accept connections. For Memcache the default is 11211, and for Redis the default port is 6379. Cannot be provided with replication_group_id.

### Actual Behavior
Not specifying a port yields an error:

```
Error: aws_elasticache_cluster.xxxxxx: "port": required field is not set
```

I'm not sure if it's the documentation that's wrong or the behavior, but the two definitely do not agree.

**Comment by catsby** [comment]
Hey @aprice  , could you share a config that reproduces this? We have a handful of nightly tests that do not specify a port and they are passing.

**Comment by bflad** [comment]
The switch from the `port` attribute being required to optional happened in #3835 and was released in [version 1.13.0 of the AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#1130-march-28-2018).

**Comment by bflad** [comment]
Closing due to lack of response -- if this is still an issue after version 1.13.0 of the AWS provider or higher, please let us know.
